system-tests/__snapshots__/results_spec.ts.js

```diff
@@ -82,6 +82,7 @@ exports['module api and after:run results'] = `
     "isInteractive": true,
     "version": "X.Y.Z",
     "protocolEnabled": false,
+    "reporterHidden": false,
```

Comment:
I'm also going to omit the internal `protocolEnabled` config option.

Reply:
> I'm also going to omit the internal protocolEnabled config option.

Good call. That shouldn't be in the results either.

Reply:
Omitted the `reporterHidden` and `protocolEnabled` options from the results. [1b0f731](https://github.com/cypress-io/cypress/pull/27482/commits/1b0f73187370483014a52e7f4636978341972570)

Note, I did not remove the keys from the [PublicConfig type](https://github.com/cypress-io/cypress/blob/6ac7febe2df80b648a733c948f4e31bd814eca37/cli/types/cypress-npm-api.d.ts#L257) since they aren't apart of the [ResolvedConfigOptions](https://github.com/cypress-io/cypress/blob/d0abe2a305e680de1349871b4905144aa99e4fdd/cli/types/cypress.d.ts#L2841).